appveyor.yml

```diff
@@ -1,21 +1,23 @@
 os: Visual Studio 2015
 
-platform:
-  - x64
-  - x86
-
 environment:
-  NODEJS_VERSION: "6"
   RUST_BACKTRACE: 1
   matrix:
-    - NODE_ARCHITECTURE: x64
+    - PLATFORM: x64
+      NODEJS_VERSION: "6"
       RUST_TOOLCHAIN: stable-x86_64-pc-windows-msvc
-
-    - NODE_ARCHITECTURE: x86
+    - PLATFORM: x86
+      NODEJS_VERSION: "6"
+      RUST_TOOLCHAIN: stable-i686-pc-windows-msvc
+    - PLATFORM: x64
+      NODEJS_VERSION: "8"
+      RUST_TOOLCHAIN: stable-x86_64-pc-windows-msvc
+    - PLATFORM: x86
+      NODEJS_VERSION: "8"
```

Comment:
This matrix looks good & looks like AppVeyor is green!
:four_leaf_clover: :four_leaf_clover: :four_leaf_clover: :four_leaf_clover: